docs/toc.md

```diff
@@ -237,7 +237,7 @@
 ### [Get Started with Visual Studio](fsharp/get-started/get-started-visual-studio.md)
 ### [Get Started with Visual Studio for Mac](fsharp/get-started/get-started-with-visual-studio-for-mac.md)
 ### [Get Started with Visual Studio Code and Ionide](fsharp/get-started/get-started-vscode.md)
-### [Get Started with Command-line Tools](fsharp/get-started/get-started-command-line.md)
+### [Get Started with with the .NET CLI](fsharp/get-started/get-started-command-line.md)
```

Comment:
We should seriously reconsider that branding @BethMassi.  Even though it comes with the SDK, it builds more than just .NET Core stuff because it's just MSBuild under the covers.